_posts/2018-10-31-cockpit-181.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Cockpit 181
+author: mvo
+date: 2018-10-31
+tags: cockpit linux
+slug: cockpit-181
+category: release
+summary: ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 181.
+
+### Graph improvements
+
+The appearance of memory and CPU graphs has been improved, and their
+colors are more accessible.
+
+![New memory graph](/images/memory-graph.png)
```

Comment:
There is an awful amount of empty white space around that PNG, can this get trimmed?

Reply:
@garrett already did this on his fork.

Reply:
Yes, also the nic editing screenshot needs some cropping.